photon-lib/py/photonlibpy/estimation/openCVHelp.py

```diff
@@ -243,7 +244,7 @@ def solvePNP_Square(
 
         # solvePnP failed
         if reprojectionError is None or math.isnan(reprojectionError[0, 0]):
-            print("SolvePNP_Square failed!")
+            logging.getLogger().error("SolvePNP_Square failed!")
```

Comment:
Should we be using the global / root logger here? I feel like no?

Reply:
Should be sorted now